raiden/tests/integration/fixtures/smartcontracts.py

```diff
@@ -31,6 +24,9 @@
     CONTRACT_USER_DEPOSIT,
 )
 
+RED_EYES_PER_CHANNEL_PARTICIPANT_LIMIT = int(0.075 * 10 ** 18)
+RED_EYES_PER_TOKEN_NETWORK_LIMIT = int(250 * 10 ** 18)
```

Comment:
As far as i can see, this is no longer used.

Reply:
Do you mean both of these constants or specifically `RED_EYES_PER_TOKEN_NETWORK_LIMIT`?

The constants `RED_EYES_PER_CHANNEL_PARTICIPANT_LIMIT` is used in a test [`DEPOSIT_FOR_TEST_API_DEPOSIT_LIMIT = RED_EYES_PER_CHANNEL_PARTICIPANT_LIMIT + 2`](https://github.com/raiden-network/raiden/pull/4824/files#diff-00c4c05ef0fac830fbdcfcb586861a27R47)

Reply:
yea i meant the `RED_EYES_PER_TOKEN_NETWORK_LIMIT`. Maybe to be able to fix the failing tests you will need this value, if that's the case then feel free to ignore this comment.